light/helpers_test.go

```diff
@@ -39,7 +43,7 @@ func genPrivKeys(n int) privKeys {
 
 // Extend adds n more keys (to remove, just take a slice).
 func (pkz privKeys) Extend(n int) privKeys {
-	extra := genPrivKeys(n)
+	extra := genPrivKeys(n,crypto.BLS12381)
```

Comment:
formatting seems off in this file.

Reply:
Do you mean add a space after the comma?

Reply:
Yes :)

Reply:
Fixed